societe-archeologique-de-bordeaux.csl

```diff
@@ -0,0 +1,304 @@
+            </group>
+            <group delimiter=", " suffix=".">
+              <text macro="title" font-style="italic"/>
+              <text variable="genre"/>
+              <text variable="publisher"/>
+              <text variable="publisher-place"/>
+              <text macro="year-date" />
+            </group>           
+          </if>
+          <else-if type="webpage">
+            <group>
+              <text macro="author"/>
+              <text macro="year-date" prefix="&#160;(" suffix=")&#160;: "/>
+            </group>
+            <group delimiter=" ">
+              <text macro="title" font-style="italic"/>
+              <text variable="URL" prefix="(" suffix=","/>
+              <text prefix=" " macro="access" suffix=")"/>
+            </group>
+          </else-if>
+          <else-if type="article-journal article-magazine article-newspaper" match="any">
+            <group>
+              <text macro="author" suffix=". "/>
+              </group>
+            <group delimiter=", " suffix=".">
+              <text macro="title" prefix="&#171; "/>
+              <text variable="container-title" font-style="italic"/>
+               <text macro="year-date" />
+		<text variable="volume"/>
+              <text variable="issue"/>             
+              <text macro="pages"/>
+            </group>
+          </else-if>
+          <else-if type="book" match="any">
+            <group>
+                <text macro="author" suffix=". "/>
+            </group>
+            <group suffix=".">
+              <text macro="title" suffix=", "/>
+              <text macro="edition"/>
+              <text macro="collection" suffix=", "/> 
+              <text variable="publisher-place" suffix=", "/>
+	     <text macro="publisher"/>
+	    <text macro="year-date" prefix=", "/>	  
+            </group>
+          </else-if>
+          <else-if type="chapter entry-dictionary" match="any">
+            <group>
+              <text macro="author" suffix=". "/>
+              </group>
+            <text macro="title" prefix="&#171; " suffix=" &#187; , "/>
+            <group>
+              <text value="in" font-style="italic" suffix="&#160;: "/>
+              <choose>
+                <if variable="editor" match="any">
+                  <text macro="editor-short-bib" />
+		<text variable="container-title" font-style="italic" prefix=", " suffix="&#160;"/>
+                </if>
+                <else>
+                  <text variable="container-title" font-style="italic" prefix=", " suffix="&#160;"/>
+                </else>
+              </choose>
+              <text macro="year-date" prefix=", "/>
+              <text macro="pages" prefix=", p. " suffix="."/>
+            </group>
+          </else-if>
+<else-if type="paper-conference" match="any">
+            <group>
+              <text macro="author" suffix=". "/>
+              </group>
+            <text macro="title" prefix="&#171; " suffix=" &#187; , "/>
```

Comment:
>use quotes="true" instead of hard-coding the guillements here and elsewhere. no because I want to have the french ones, not the english ones

quotes="true" in this style will use French guillemets. Try it out.